README.md

```diff
@@ -125,12 +125,15 @@ names to the aliases.
 We use bower to manage our javascript dependencies. In order to download the required javascript packages,
 you'll need to run `./manage.py bower install` and install `bower`. Follow these steps to install:
 
-1. If you do not already have it, install [npm](https://www.npmjs.com/). In Ubuntu this is now bundled
-with NodeJS. An up-to-date version is available on the NodeSource repository.
+1. If you do not already have npm:
```

Comment:
I think you need to leave a blank line here if you want a visible line break in the README.  Here's what this looks like:
https://github.com/dimagi/commcare-hq/blob/938ff4ac4228b92e0dd247097d92486720bfb5e9/README.md#installing-bower